source/dynk.f90

```diff
@@ -183,8 +183,11 @@ subroutine dynk_parseInputLine(inLine,iErr)
     write(lout,"(a)") "DYNK> Debugging is ENABLED"
 
   case("NOFILE")
-    dynk_noDynkSets = .true.
-    write(lout,"(a)") "DYNK> Disabled writing '"//dynk_fileName//"'"
+    write(lout,"(a)") "DYNK> The NOFILE flag no longer has any effect"
```

Comment:
This should maybe be an error?

Reply:
No, it will not be made an error. The dynkjsets file explains why it's empty.

Reply:
No, it will not be made an error.

When asking for NOFILE, you get exactly what you ask for since NOFILE is now default behaviour. Error on this is just unnecessarily breaking input files that is asking for exactly what they're getting.